docs: update images and add 'classic session launcher' description (#92)
lablup/backend.ai-docs-webui: docs/user_settings/user_settings.rst

   WebUI is open sourced, anyone who willing to make the translation better
   can contribute: https://github.com/lablup/backend.ai-webui.

Automatic Update Check
^^^^^^^^^^^^^^^^^^^^^^

> A notification window pops up when a new version of the Web-UI is detected.

Reviewer: ```suggestion
A notification window pops up when a new WebUI version is detected.
```

It works only in an environment where Internet access is available.

Auto logout
^^^^^^^^^^^

> Log out automatically when all Backend.AI Web-UI pages are closed except for

Reviewer: ```suggestion
Log out automatically when all Backend.AI WebUI pages are closed except for
```

   :width: 400
   :align: center

Classic Session Launcher
^^^^^^^^^^^^^^^^^^^^^^^^

> From Backend.AI version 24.09, Backend.AI offers an improved version of the session launcher (NEO) as default.

Reviewer: ```suggestion
From version 24.09, Backend.AI offers an improved version of the session launcher (NEO) as default.
```
